js_tests/tests.html

```diff
@@ -78,6 +78,13 @@
     <script src='../django/contrib/admin/static/admin/js/prepopulate.js' data-cover></script>
     <script src='../django/contrib/admin/static/admin/js/urlify.js' data-cover></script>
 
+    <div id="id_point_map">
+        <textarea id="id_point" class="vSerializedField required" style="display:none;" name="point" rows="10" cols="150">POINT (7.8177 47.397)</textarea>
+    </div>
+    <script src='http://openlayers.org/api/2.13/OpenLayers.js' data-cover></script>
```

Comment:
Yes, if we did I'd think it'd be in `js_tests`. I don't have a strong opinion -- maybe Trey will have some input.